### PR TITLE
make permissions available to the artifactory object directly.

### PR DESCRIPTION
**Pull request by dogfish182**

### CODE REVIEW

File: pyartifactory/objects.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def __init__(
         self.security = ArtifactorySecurity(self.artifactory)
         self.repositories = ArtifactoryRepository(self.artifactory)
         self.artifacts = ArtifactoryArtifact(self.artifactory)
+        self.permissions = ArtifactoryPermission(self.artifactory)
```
Comment (anancarv) [inline pyartifactory/objects.py]: OMG this is a huge oversight from the latest release, thank you for pointing it out :pray:

**Comment by anancarv** [comment]
Since it's the patch of the library, could you also increase the lib version by running the following command:
- **poetry version patch** (will increase the package version to 1.1.0)
- edit the file **pyartifactory/__init_.py** by increasing the version to 111 as well

Then I'll just have to make a new release for publishing your patch

**Comment by nymous** [comment]
About that @anancarv, I don't think feature branches should bother with updating the version. If you merge multiple branches before making a release, everyone has to bump the version to meaningless numbers? ^^ I'd rather do this in the master branch right as you cut the new release, prepare the changelog, and tag this commit.

**Comment by anancarv** [comment]
Seems logic. Ok, I'll accept thisPR then and make a new one for increasing the app version

**Pull request merged**